R/PseudoBulk.R

```diff
@@ -1107,6 +1107,7 @@ PseudobulkingDEHeatmap <- function(seuratObj, geneSpace = NULL, contrastField =
 #' @param devianceCutoff Tolerance for model error when deciding how much regularization should be performed. 1 = no tolerance for error, 0 = intercept only, no genes used for prediction.
 #' @param split the option to provide a previous model's training/testing set. This is necessary if you're performing multiple iterations of model fitting. 
 #' @param returnModelAndSplits A boolean option to return a list containing the fitted model and training/testing splits in addition to the useful features. 
+#' @param excludeVariableGenes A boolean option to exclude variable genes from the analysis. Only the Variable Gene Exclusion (v2) list from RIRA is supported.
```

Comment:
see comment below about making this a gene list with a default, rather than boolean